versions.properties

```diff
@@ -29,7 +29,7 @@ version.com.nhaarman.mockitokotlin2..mockito-kotlin=2.1.0
 version.google.android.material=1.2.1
 version.google.dagger=2.27
 version.io.reactivex.rxjava2..rxandroid=2.0.2
-version.io.reactivex.rxjava2..rxjava=2.1.10
+version.io.reactivex.rxjava2..rxjava=2.2.9
```

Comment:
Bumped this dependency bc I realised that it was overriden by the `AndroidX.work.rxJava2`, which was bringing version 2.2.9 along, so we effectively had two rxjava versions in the APK, but we were using 2.2.9